bmaptools/TransRead.py

```diff
@@ -664,12 +664,10 @@ def _print_warning(timeout):
 
     def read(self, size=-1):
         """
-        Read the data from the file or URL and and uncompress it on-the-fly if
+        Read the data from the file or URL and uncompress it on-the-fly if
```

Comment:
Hi, thanks for working on this.
Could you please split this typo fix into its own commit?

Reply:
Yes, should it go in its own PR, or can I add the commit here?

Reply:
Whichever is easier for you. Thanks!